Core/Core/API/APIRequestable.swift

```diff
@@ -41,6 +42,8 @@ public enum APIQueryItem: Equatable {
             return APIQueryItem.array("include", includes).toURLQueryItems()
         case .perPage(let perPage):
             return APIQueryItem.value("per_page", String(perPage)).toURLQueryItems()
+        case .bool(let name, let value):
+            return APIQueryItem.value(name, value ? "1" : "0").toURLQueryItems()
```

Comment:
Do all canvas API's take Bool as a 1 or 0?  Or can it be "true" or "false" sometimes?

Reply:
It does support true/false as well.  Also yes/no, y/n, t/f, on/off. I figure we might as well be consistent.

Here's how the api parses boolean params:
```ruby
    def self.value_to_boolean(value, ignore_unrecognized: false)
      if value.is_a?(String) || value.is_a?(Symbol)
        return true if ["yes", "y", "true", "t", "on", "1"].include?(value.to_s.downcase)
        return false if ["no", "n", "false", "f", "off", "0"].include?(value.to_s.downcase)
      end
      return value if [true, false].include?(value)
      return nil if ignore_unrecognized
      return value.to_i != 0
    end
  end
```

Reply:
ah ok, yes this works well then.  good job.